ipatests/test_integration/test_membermanager.py

```diff
@@ -212,3 +212,75 @@ def test_hostgroup_member_manager_nopermission(self):
             "'write' privilege to the 'memberManager' attribute of entry"
         )
         assert expected in result.stdout_text
+
+    def test_member_manager_upgrade_scenario(self):
+        """
+        Testing if manager whose rights defined by the group membership
+        is able to add group members, after upgrade of ipa server.
+        Using ACI modification to demonstrate unability before upgrading
+        ipa server.
+
+        Related: https://pagure.io/freeipa/issue/9286
+        """
+        user = "idmuser"
+        password = "Secret123"
+        group1 = "role-groupmanager"
+        group2 = "role-usergroup-A"
+
+        master = self.master
+
+        tasks.kinit_admin(master)
+        tasks.group_add(master, group1)
+        tasks.group_add(master, group2)
+        tasks.create_active_user(master, user, password)
+
+        tasks.kinit_admin(master)
+        tasks.group_add_member(master, group1, user)
+        master.run_command(["ipa", "group-add-member-manager", "--groups",
+                            group1, group2])
+
+        base_dn = self.master.domain.basedn
+        aci_hostgroup = (
+            '(targetattr = "member")(targetfilter = '
+            '"(objectclass=ipaHost Group)")'
```

Comment:
You need to remove the space between ipaHost and Group. The line should look like:
```
            '"(objectclass=ipaHostGroup)")'
```

Reply:
Yes, it was a typo caused by rewriting the aci. Thank you for correction.